.buildkite/pipeline.yml

```
@@ -15,227 +15,14 @@ plugins: &plugins
     mode: checkout
 
 steps:
-  - label: ":docker::python: build 3.6 runner"
+  - label: ":python: test Conda build"
     <<: *timeout
-    key: "runner-3_6"
+    command: ".buildkite/steps/conda-build.sh"
```

This script is one command, so, from my perspective, it would be fine to do this as 

```suggestion
    command: "conda build . --no-anaconda-upload"
```